lib/mongo/client.rb

```diff
@@ -116,7 +116,8 @@ class Client
     # @since 2.5.0
     VALID_COMPRESSORS = [
       Mongo::Protocol::Compressed::SNAPPY,
-      Mongo::Protocol::Compressed::ZLIB
+      Mongo::Protocol::Compressed::ZLIB,
+      Mongo::Protocol::Compressed::ZSTD
```

Comment:
Let's have zstd ahead of snappy here also and please add the trailing comma to the last entry.